extension/indexer/accepted_subscriber.go

```diff
@@ -0,0 +1,31 @@
+// Copyright (C) 2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package indexer
+
+import (
+	"context"
+
+	"github.com/ava-labs/hypersdk/chain"
+)
+
+type AcceptedSubscriber interface {
+	Accepted(ctx context.Context, blk *chain.StatelessBlock) error
+}
+
+type AcceptedSubscribers struct {
+	subscribers []AcceptedSubscriber
+}
+
+func NewAcceptedSubscribers(subscribers ...AcceptedSubscriber) *AcceptedSubscribers {
+	return &AcceptedSubscribers{subscribers: subscribers}
+}
+
+func (c *AcceptedSubscribers) Accepted(ctx context.Context, blk *chain.StatelessBlock) error {
```

Comment:
nit: update to `a` since we updated this struct name

Reply:
Done